src/pages/news/gsoc-2023-eclipse-gradle-kotlin/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Google Summer of Code 2023 - Eclipse Gradle Kotlin Script Support"
+date: 2023-12-18 10:00
+---
+
+The Kotlin Foundation sponsored 4 projects for the Google Summer of Code 2023, and we are excited to announce the completion of the Eclipse Gradle Kotlin Script Support project! Gradle, a Kotlin Foundation member, provided mentoring support for this project, and we'd like to share details about the project.
+
+The contributor to this project was [Nikolai Vladimirov](https://www.linkedin.com/in/vladimir0v/), a third-year student from Constructor University and the mentor was [Donát Csikós](https://github.com/donat), a Senior Software Engineer at Gradle.  The project aimed to bring support for Gradle Kotlin Scripts (KTS) to the Eclipse IDE.
+
+The initial goal was to improve user interaction with Gradle Kotlin scripts inside the Eclipse IDE. The goal was to implement the following functionality:
+
+- Syntax highlighting for Kotlin scripts
+- Content assistance
+- Go-to-definition
+- Accurate diagnostics for syntax errors
+- Signature help
+- Hovering
+
+Nikolai successfully delivered these features by implementing support for Kotlin scripts based on the existing kotlin-language-server project, creating a new system to separate compilation environments for each build script.  Here are examples of each feature:
+
+**Syntax highlighting for KTS scripts**
+
+![Syntax highlighting for KTS scripts](image5.png)
+
+**Content assistance**
+
+![Content assistance](image1.png)
+
+**Go-to-definition**
+
+![Go-to-definition](image4.gif)
```

Comment:
There is support only `jpg`, `png`, `webp`, `tiff`, `avif` formats 😞
Is it possible to get better quality image in this place?
<img width="345" alt="Screenshot 2023-12-18 at 12 50 27" src="https://github.com/KotlinFoundation/kotlinfoundation.github.io/assets/242514/c3b35b3d-d873-4310-a572-8b12112b9605">